serverlessworkflow/antora.yml

```diff
@@ -24,16 +24,4 @@ asciidoc:
     open_api_spec_url: https://spec.openapis.org/oas/v3.1.0.html
     quarkus_openapi_gen_url: https://github.com/quarkiverse/quarkus-openapi-generator
     kie_tools_releases_page_url: https://github.com/kiegroup/kie-tools/releases
-    java_install: https://www.java.com/en/download/help/download_options.html
-    maven_install: https://maven.apache.org/install.html
-    docker_install: https://docs.docker.com/engine/install/
-    podman_install: https://docs.podman.io/en/latest/
-    kubectl_install: https://kubernetes.io/docs/tasks/tools/install-kubectl
-    java_install_url: https://www.java.com/en/download/help/download_options.html
-    maven_install_url: https://maven.apache.org/install.html
-    docker_install_url: https://docs.docker.com/engine/install/
-    podman_install_url: https://docs.podman.io/en/latest/
-    kubectl_install_url: https://kubernetes.io/docs/tasks/tools/install-kubectl
-    kn_cli_install_url: https://github.com/knative/client/blob/main/docs/README.md#installing-kn
-    kafka_doc_url: https://kafka.apache.org/documentation/ 
```

Comment:
@hmanwani-rh I believe we need these variables since they are all very common to our use cases.

Reply:
Done, fixed it.